src/conv/invokers/impl_gemm_dynamic.cpp

```diff
@@ -241,13 +241,24 @@ InvokerFactory MakeImplGemmDynamicBackwardDataInvokerFactory(const ConvolutionCo
         const int gemm_k_gid = k * y_dot_slice_gid[gemm_id] * x_dot_slice_gid[gemm_id];
         is_gemm_not_empty.emplace_back(gemm_k_gid > 0);
     }
+    bool need_set_zero = false;
+    if(y < stride_h || x < stride_w)
```

Comment:
Dilation is also a factor. I don't have an exact logic to decide when to set-zero, but it's safe to od

```
if(!(dilation_h == 1 && dilation_w==1 && y >=  stride_h && x >= stride_w))
    need_set_zero = true;
```

Reply:
Oh, yes, dilation also have same effect. Let me change it